lib/build.js

```diff
@@ -2,7 +2,9 @@ const fs = require("fs")
 const nodePandoc = require("node-pandoc-promise")
 const os = require("os")
 const path = require("path")
-
+const stylesheet = require("./stylesheet")
```

Comment:
Is it just to make the code lighter that you replace the default `assets/stylesheets.css` with a js file, or is there another reason?